protostar/commands/deploy_command.py

```diff
@@ -38,7 +39,7 @@ def example(self) -> Optional[str]:
         return "protostar deploy ./build/main.json --network alpha-goerli"
 
     @property
-    def arguments(self) -> List[Command.Argument]:
+    def arguments(self):
         return [
             Command.Argument(
```

Comment:
Why not use ProtostarArgument everywhere to be consistent?

Reply:
I want to keep the PR small. I'll do it later.

Reply:
that's fine